docs/api/hardhat/getting-started.md

```diff
@@ -78,20 +81,18 @@ module.exports = {
 ```
 
 ::: tip
-
-To learn more about each specific property in the `hardhat.config.ts` file, check out the [plugins documentation](./plugins.md)
-
+To know more about the individual properties in the `hardhat.config.ts` file, you can refer to the [plugins documentation](./plugins.md).
 :::
 
 ## Write and deploy a contract
 
-3. Create the `contracts` and `deploy` folders. In the `contracts` folder we will store all the smart contract files. In the `deploy` folder we'll place all the scripts related to deploying the contracts.
+3. Create two folders: `contracts` for storing smart contract files, and `deploy` for keeping scripts related to deploying the contracts.
 
-4. Create the `contracts/Greeter.sol` contract and paste the following code:
+4. Create a new contract file called `Greeter.sol` in the `contracts` folder, and copy/paste the following code into it:
 
 ```solidity
 //SPDX-License-Identifier: Unlicensed
-pragma solidity ^0.8.0;
+pragma solidity ^0.8.17;
```

Comment:
Change to the solc version you are using in hardhat config. They need to match.

Reply:
They match.